### DISCUSSION TITLE
A question regarding removing key prop

### DISCUSSION THREAD

**Discussion by blazk0** [Shopify/flash-list#693] [Q&A] [answered]
In the [Docs](https://shopify.github.io/flash-list/docs/fundamentals/performant-components#remove-key-prop), It says that its better to remove the `key prop`. 

What about the `keyExtractor prop` in `FlashList` itself, should we avoid using it ?

**Comment by fortmarek** [comment, marked as answer]
`keyExtractor` is not passed as a `key` prop, so you can safely use it. In some cases, it can actually lead to perf benefits, as mentioned in the [docs](https://shopify.github.io/flash-list/docs/usage#keyextractor). It's ok if you use it, although usually it should not be necessary.
